controller/DonationController.js

```diff
@@ -41,6 +42,46 @@ exports.makeDonation = catchAsyncError(async (req, res) => {
   }
 });
 
+/**
+ * @description Donate larger sums of money (NGN 500,001 and upwards)
+ * @route POST /v1/donation/bulk-sum
+ * @access PUBLIC
+ */
+exports.createManualDonation = catchAsyncError(async (req, res) => {
```

Comment:
@Chiemezuo, why is this showing as a new addition. Did you create a branch off `develop` to make these changes?